src/audio/sound_manager.cpp

```diff
@@ -394,13 +396,15 @@ SoundManager::set_sound_volume(int volume)
 void
 SoundManager::resume_music(float fadetime)
 {
+  
   if (m_music_source == nullptr)
     return;
 
   if (fadetime > 0) {
     if (m_music_source
-       && m_music_source->get_fade_state() != StreamSoundSource::FadingResume)
-      m_music_source->set_fading(StreamSoundSource::FadingResume, fadetime);
+       && m_music_source->get_fade_state() != StreamSoundSource::FadingResume){
+        m_music_source->set_fading(StreamSoundSource::FadingResume, fadetime);
+       }
```

Comment:
Same as above

Why have these lines changes? They don't seem related to the PR...